src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/NamespaceCache.java

```diff
@@ -20,9 +24,11 @@ public class NamespaceCache {
 
   private final ObjectMapper objectMapper;
 
+  // TODO: move the settings to config
+  // TODO: set the cache loader when creating the cache
   private static final long CACHE_TTL_SECONDS = 300;
   private static final long CACHE_MAX_SIZE = 1000;
-  private final Cache<String, CollectionSettings> vectorCache =
+  private final Cache<String, SchemaObject> schemaObjectCache =
```

Comment:
So we have to use `SchemaObject` here because they can be `CollectionSchemaObject`s or `TableSchemaObject`s?

I wonder if there should be common base class for those 2 (`ContainerSchemaObject`?) since we often have the case of "one of these 2 SchemaObject types.... may have shared settings, compared to Namespace- and DatabaseSchemaObjects.

Reply:
Possible future change, skipping for now.